examples/03-transform-any.php

```diff
@@ -36,6 +33,8 @@
         // demo result includes full user from NDJSON with additional properties
         $user['result'] = $body;
         return $user;
+    }, function (Exception $e) {
+        echo 'Error: ' . $e->getMessage() . PHP_EOL;
```

Comment:
See https://github.com/clue/reactphp-flux/pull/30#discussion_r1527145043

```suggestion
```

Reply:
I changed it thank you @clue. I also decided to change it in the `examples/02-transform-all.php` file for the same reason.